packages/eks-controller/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Assuming we should do apiVersion 2 for helm 3 only, otherwise we might get stuck trying to support helm 2 down the line.

Reply:
Nvm, this will be helm 2